src/util/XDRStream.h

```diff
@@ -23,7 +25,8 @@ class XDRInputFileStream
 {
     std::ifstream mIn;
     std::vector<char> mBuf;
-    unsigned int mSizeLimit;
+    std::streamsize mSizeLimit;
```

Comment:
I'd rather be consistent: let's use `size_t` everywhere instead of stream specific types